main/zoe/api/Zoe-Intro.md

```diff
@@ -0,0 +1,109 @@
+
+
+While there can be more than one Zoe instance installed and running at any time, for now we’ll focus on the single instance running on Agoric’s testnet. Of course, before you can run a smart contract on Zoe, you have to install it there. For Zoe purposes, *installation* means installing the code your contract is written in (JavaScript and of course the ERTP and Zoe API commands we’ve defined) on a Zoe instance. 
+
+
+Contract installation returns an `installationHandle`, a unique identifier for the installed contract used to create *contract instances* from the installed code. A contract installation is a way to reuse the same code multiple times and ensure that widely used contract code is recognizable under the same identity. For example, if I negotiate with someone and we agree to use the “Atomicswap” contract **tyg todo link**, if I make an instance of that contract, my counterparty can use its `installationHandle` with the `getInstallation()` API method to check the code is the `Atomicswap` code they expected.
+
+
+## Running contracts on Zoe
+
+
+First, use `zoe.makeInstance()` to create an instance of the installed contract you want to use. The method takes as arguments:
+- the `installationHandle` of the specific contract to use.
+- a *keyword record* of name : value pairs.
+  - The names are words used in the contract, such as “Asset” or “Bid” that, in the contract, take an amount as its value. 
+  - The values are references to an *issuer* (see Introduction to ERTP **tyg todo: add link) of the type you want the associated name to accept as its value in the contract instance. For example, if your contract uses the name “Asset” for what the offer in the contract is (e.g. 3) and the resource unit is quatloos, and uses “Price” for the counter-offer, which is in moola, the keyword record value would be ((“Asset” : quatlooIssuer) (“Price” : moolaIssuer)).
+- *terms*: Another set of key : value pairs.
+  -   These customize contract aspects expressed in the contract without specific values. For example, an auction contract’s may not define what the minimum bid has to be or the minimum amount one has to raise from the current high bid, just that there are such values which the contract uses. These values are different for a collection of used Harry Potter paperbacks and a signed first British hardback edition of *Harry Potter and the Philosopher’s Stone*. If you write code in the contract to access that instance’s `terms`’ name:value pairs of `('MinBid' : 25, 'Currency : 'USDollar', 'MinRaise' : 5)` or  `('MinBid' : 100000, 'Currency : 'USDollar', 'MinRaise' : 5000)`, you’ve customized the generic contract for that specific instance. 
+
+
+`zoe.makeInstance()` returns an *invite* to be a party in that contract instance. Thus, the party who made the contract instance has an invite to it. The `instanceHandle`, the unique identifier for the contract instance, can be retrieved from the invite using `GetInstanceHandle()`.
```

Comment:
Should show arguments of `zoe~.makeInstance(...)` (or `E(zoe).makeInstance(...)`). Show the arguments where I said "`...`" above.

Reply:
I'll think about this; in at least some cases for an intro, I just want them to know what's happening and what the request to do it is, putting off the more detailed arguments a bit. 

Quick question; are the calls in fact "zoe~." instead of "zoe."? I don't recall seeing the ~ version, certainly not in the existing docs.

Reply:
We shouldn't use ~ currently in the docs, since we are not recommending that our users use it until they can use it with VSCode and not get linting errors.

Reply:
If we're not using `~.`, then it should be
```js
E(zoe).makeInstance(...)
```
It should never be
```js
zoe.makeInstance(...)
```
If it is in our other docs, those need to be corrected to. There are no realistic circumstances in which the user can write the second and have it work.